GOVERNANCE.md

```diff
@@ -9,49 +9,72 @@ In order to provide equitable rights to all Working Group members,
+Each member identifies their affiliation with an organization, company or
+open source project, along with their specific use case(s) for events.
+
+- company / org / project 
+    - representative, (optional) alternate representative
+    - describe how you are using or intend to use the Event specification
+    
+**Active Members** participate in at least three Working Group meetings per month.
+
 ## PRs
 
-Typically, PRs are expected to meet the following criteria prior to being
+Changes fall into two categories:
+- **Administrative**: grammar, spelling, minor clarifications, or git rebase.  These must be left open for at least 24 hours, then they may be merged by someone who is not the author (or +1 from a member representing a different company or organization). All comments must be addressed before merging with additional +1 once any comment has been addressed.
+- **Substantive**: modifications to the name or definition of a term or additions to the specification.  When in doubt, committers should assume that a PR is substantive and leave open until next WG meeting.
+
+PRs are expected to meet the following criteria prior to being
 merged:
 
 * The author of the PR indicates that it is ready for review by asking for it
   to be discussed in an upcoming meeting - eg. by adding it to the agenda
-  document.
+  document.  If the author has become inactive, then another WG member can 
+  volunteeer to take on this role.
 * All comments have been addressed.
 * PRs that have objections/concerns will be discussed off-line by interested
   parties. A resolution, updated PR, will be expected from those talks.
+* Resolving PRs ("merging" or "closing with no action") will be done as a
+  result of a motion made via comment at least two days in advance of a 
+  WG meeting, and approved at that meeting.
+* Reopening a PR can be done if new information is made available via
+  github issue, which can then be added to the agenda for WG meeting.
+
+## Decision Making
+
+Process:
+- [Optional] Add an item to the [agenda]() for a working group meeting if you have an idea or suggestion and want some feedback before drafting a proposal. If the WG has a busy agenda, then you may be asked to convene a group of interested parties separately from the WG meetings.
+- Proposals, added via PR to this github repo, must be open for at least a week, and will not be considered for decision if substantive changes were made within two days of a meeting
+
+Most of the time we come to agreement simply by revising a proposed change or addition until all of the issues raised are addressed.  Occasionally, the Working Group may choose to overrule a minority viewpoint by calling for a vote by active members of the Working Group, as described below.  
 
-## Voting
+If a minority viewpoint is supported by an member with an implementation, the vote cannot be held until there is an open source implementation that illustrates the benefits or drawbacks of the proposal.  
```

Comment:
Can the member's implementation be closed-source? Probably worth clarifying this bar.

Reply:
an -> a